livingatlas/pipelines/src/main/java/au/org/ala/pipelines/java/ALAVerbatimToInterpretedPipeline.java

```diff
@@ -373,13 +379,13 @@ public static void run(InterpretationPipelineOptions options, ExecutorService ex
 
     log.info("Saving metrics...");
     MetricsHandler.saveCountersToTargetPathFile(options, metrics.getMetricsResult());
-    log.info("Pipeline has been finished - {}", LocalDateTime.now());
+    log.info("Pipeline has been finished - " + LocalDateTime.now());
```

Comment:
Isn't the {} form supposed to be more efficient, since it doesn't render the argument if logging is not needed?

Reply:
yep. ill revert